src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,165 @@
 package org.example;
 
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Main {
+
+    private static String[][] tasks;
+    private static Scanner scanner;
+    static final String[] OPTIONS = {"add", "remove", "list", "exit"};
+
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        scanner = new Scanner(System.in);
+        tasks = readTasks();
+
+        while (true) {
+            displayUserOptions();
+            switch (scanner.nextLine().toLowerCase()) {
+                case "add" -> tasks = addTask();
+                case "remove" -> tasks = removeTask();
+                case "list" -> listTasks();
+                case "exit" -> exitTasks();
```

Comment:
CLEAN-CODE: lepsi pojmenovani metody.
```suggestion
                case "exit" -> exit();
```

Reply:
Opraveno.